### PR TITLE
Pass all admin to lang, and do users settings view and logic.

### PR DESCRIPTION
**Pull request by srestrep74**
# Description

I pass all my admin views to lang messages, and make the view and controller of users settings page.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lang/en/admin.php
Comment (JuanM0412) [inline lang/en/admin.php:1]: Admin comment must be replaced for category (line 44)

**Review by JuanM0412 — APPROVED** [review]

File: app/Http/Controllers/UserController.php
Comment (LuisaMG01) [inline app/Http/Controllers/UserController.php:1]: The flash comment should be inside the lang and not as static text inside the controller

File: lang/en/admin.php
Comment (LuisaMG01) [inline lang/en/admin.php:1]: Spacing differs on line 95, leave a space under the comment also

File: lang/en/app.php
Comment (LuisaMG01) [inline lang/en/app.php:1]: Spacing differs in the admin lang and the app, a space must be left before and after the comment

**Review by LuisaMG01 — APPROVED** [review]

**Review by mahoyos — APPROVED** [review]

**Pull request merged**